install.sh

```diff
@@ -666,6 +703,7 @@ Restart=always
 RestartSec=5s
 ExecStartPre=-/sbin/modprobe br_netfilter
 ExecStartPre=-/sbin/modprobe overlay
+ExecStartPre=-/sbin/modprobe iptable_nat
```

Comment:
modprobes have been moved to [`syssetup/setup.go` ](https://github.com/rancher/k3s/blob/ba70c41cce76fcae70867754cdf390501a8cef1b/pkg/agent/syssetup/setup.go)